### ISSUE TITLE
Unable to delete Report

### ISSUE THREAD

**Issue by dan-m-joh** [sni/Thruk#1095]
While trying Thruk 2.42.1 (from ConSol Labs repository) I get the following error message when I try to delete a newly created  Report:

![image](https://user-images.githubusercontent.com/26163222/115701645-4a18d600-a368-11eb-9469-034f09f2479f.png)

We are using "Basic" authentication - if that matters.

 - Browser: Firefox 78.10.0esr (64-bit)

Regards,
Dan

**Comment by dan-m-joh** [comment]
And you have a typo (I believe) in "lib/Thruk/Controller/error.pm" for error '24':

```
--- error.pm    2021-04-23 15:53:33.862315665 +0200
+++ error.pm.org        2021-04-23 15:52:49.403652818 +0200
@@ -213,8 +213,8 @@
             'code' => 500, # internal server error
         },
         '24'  => {
-            'mess'    => 'CSRF Security Alert',
-            'dscr'    => 'Using this formular requires a POST with a valid CSRF token or an API key.',
+            'mess'    => 'CSFR Security Alert',
+            'dscr'    => 'Using this formular requires a POST with a valid CSFR token or an API key.',
             'code'    => 403, # forbidden
             'log_req' => 1,
         },
```

It does not help me with my error, but I just wanted to tell.

-- 
Dan

**Comment by sni** [comment]
You are right. Should be fixed with 0b849a60ac96ae37225928a000b8283066ef0694

**Comment by dan-m-joh** [comment]
Great, now it works.

Thanks and have a nice Weekend!
